src/server/pfs/server/master.go

```diff
@@ -44,10 +43,19 @@ func (d *driver) master(ctx context.Context) {
 		eg.Go(func() error {
 			return d.storage.GC(ctx)
 		})
-		eg.Go(func() error {
-			gc := chunk.NewGC(d.storage.ChunkStorage())
-			return gc.RunForever(ctx)
-		})
+		if d.env.StorageConfig.StorageGCPolling == "-1" {
+			// skipping GC is useful for potentially rolling back to a previous state
+			log.Info("Skipping initialization of Chunk GC because STORAGE_GC_POLLING set to -1")
+		} else {
+			gcPeriod, err := time.ParseDuration(d.env.StorageConfig.StorageGCPolling)
```

Comment:
This is a little late in the initialization to parse this.   This errors the master, but then it will restart, only to error again.

I think this should be integer seconds so we can rely on the env parsing stuff to fail if it can't be parsed into an integer.  Above we should check for < 0.  So all integer values are allowed.